http/http_client.hpp

```diff
@@ -783,15 +784,16 @@ class ConnectionPool : public std::enable_shared_from_this<ConnectionPool>
         }
         else if (requestQueue.size() < maxRequestQueueSize)
         {
-            BMCWEB_LOG_ERROR << "Max pool size reached. Adding data to queue."
-                             << destIP << ":" << std::to_string(destPort);
+            BMCWEB_LOG_CRITICAL
+                << "INFO: Max pool size reached. Adding data to queue."
+                << destIP << ":" << std::to_string(destPort);
             requestQueue.emplace_back(std::move(thisReq), std::move(cb));
         }
         else
         {
             // If we can't buffer the request then we should let the callback
             // handle a 429 Too Many Requests dummy response
-            BMCWEB_LOG_ERROR << destIP << ":" << std::to_string(destPort)
+            BMCWEB_LOG_ERROR << destIP << ": " << std::to_string(destPort)
```

Comment:
Is a blank really wanted between the ip and the port here?

Reply:
yes , i think for better readibilty